build.cake

```diff
@@ -41,6 +42,7 @@ var IMAGE_DIR = PROJECT_DIR + "images/";
 var MSI_DIR = PROJECT_DIR + "msi/";
 var CURRENT_IMG_DIR = IMAGE_DIR + $"NUnit-{productVersion}/";
 var CURRENT_IMG_NET20_BIN_DIR = CURRENT_IMG_DIR + "bin/net20/";
+var CURRENT_IMG_AGENTS_BIN_DIR = CURRENT_IMG_DIR + "bin/agents";
```

Comment:
```suggestion
```

Looks like this was unused in the end. 🙂

Reply:
Yes, it was a remnant of my earlier proposal to change the structure under BIN_DIR so that it would match the intended package structure.